pkg/registry/bundle_test.go

```diff
@@ -0,0 +1,108 @@
+package registry
+
+import (
+	"io/ioutil"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime/serializer"
+	k8syaml "k8s.io/apimachinery/pkg/util/yaml"
+	"path/filepath"
+	"reflect"
+	"strings"
+	"testing"
+
+	apiextensionsv1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
+	apiextensionsv1beta1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1beta1"
+)
+
+const (
+	manifestDir = "./testdata/v1crd_bundle/manifests"
+)
+
+// TestV1CRDsInBundle tests that adding a v1 and v1beta1 CRD to a bundle is successful.
+// The provided APIs and CRD objects in the created bundle are compared to those in a test manifest directory.
+func TestV1CRDsInBundle(t *testing.T) {
```

Comment:
I think it might be simpler to tweak an existing test -- say in `populator_test.go` -- to contain a v1 CRD.

Reply:
It could but since I'm changing `bundle.go` I wanted to write a `bundle_test.go`. Frankly surprised we didn't have unit tests there already.